docs/docs/config-reference.md

```diff
@@ -255,6 +255,17 @@ This setting is useful for stripping HTML comments created by PR templates.
 
 This option only applies when `merge.message.body_type = "markdown"`.
 
+### `merge.message.cut_body_before`
```

Comment:
The docs deploy immediately on merge, so usually add docs in a separate PR that I merge after releasing a new version of the bot.

Reply:
I reverted the docs commit in this PR. I will probably open a new PR with the content of the docs. :slightly_smiling_face: